src/main/java/com/example/myhealthblock/style/annotation/UserController.java

```diff
@@ -0,0 +1,29 @@
+package com.example.myhealthblock.style.annotation;
+
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Mono;
+
+@RestController
+@RequestMapping("/annotation/user")
+public class UserController {
+
+    @GetMapping("/{id}")
+    public Mono<String> getUser(@PathVariable String id) {
+        return Mono.just("User ID: " + id);
+    }
+
+    @PostMapping
+    public Mono<String> createUser() {
+        return Mono.just("User created");
+    }
+
+    @PutMapping("/{id}")
+    public Mono<String> updateUser(@PathVariable String id) {
+        return Mono.just("User ID: " + id + " updated");
+    }
+
+    @DeleteMapping("/{id}")
+    public Mono<String> deleteUser(@PathVariable String id) {
+        return Mono.just("User ID: " + id + " deleted");
```

Comment:
추후 디버깅 시 .log()를 통해 동작 흐름을 파악해 보는 것도 좋을 것 같군요!
수고하셨습니다 :)
```suggestion
        return Mono.just("User ID: " + id + " deleted").log();
```